### PR TITLE
Refresh and improve the tracking documentation

### PR DESCRIPTION
**Pull request by nirvn**
Improvements include:
- Addition of details on tracking against a point layer (which was not possible until now)
- New rounds of screenshots feature up-to-date QField 2.3 looks
- A fair amount of rewording, which I hope improves things a bit

### CODE REVIEW

File: documentation/how-to/tracking.en.md [hunk 1]
```diff
@@ -4,32 +4,65 @@ title: Tracking
...
-needs to be set.
+To activate tracking, you must first make sure the positioning is active. Then,
+open the side dashboard, long-press on a legend layer within which you want to
+save your tracks to, and select the *Setup tracking* button to configure
+the tracking session.
+
+!![](../assets/images/tracking-layer-properties.png)
+
+Two constraints are available to limit the number of vertices recorded
+during tracing:
```
Comment (m-kuhn) [inline documentation/how-to/tracking.en.md:23]:
```suggestion
during tracking:
```

File: documentation/how-to/tracking.en.md [hunk 2]
```diff
@@ -4,32 +4,65 @@ title: Tracking
...
+- A minimum time interval; and
```
Comment (m-kuhn) [inline documentation/how-to/tracking.en.md:24]:
```suggestion
- A minimum time interval
```

File: documentation/how-to/tracking.en.md [hunk 3]
```diff
@@ -4,32 +4,65 @@ title: Tracking
...
+- A minimum distance.
+
+!![](../assets/images/tracking-settings.png)
+
+Once you have finished configuring your tracing session, hit the *Start tracking*
```
Comment (m-kuhn) [inline documentation/how-to/tracking.en.md:29]:
```suggestion
Once you have finished configuring your tracking session, hit the *Start tracking*
```

File: documentation/how-to/tracking.en.md [hunk 4]
```diff
@@ -4,32 +4,65 @@ title: Tracking
...
+you to define the attributes of the feature(s) that will be created while QField
+tracks your position.
+
+For line and polygon layers, a single feature - its geometry formed of vertices
+from the recorded positions - will be create per tracking session.  For point
+layers, a feature will be created for each recorded position, with attribute
+values entered in the form remembered across features.
+
+!!! note
+    QField will skip the feature form step if the layer is configured to hide all attributes. This can streamline the process of starting tracking sessions.
```
Comment (m-kuhn) [inline documentation/how-to/tracking.en.md:40]: ... or if the form has been configured to be hidden in the attribute form settings

image

File: documentation/how-to/tracking.en.md [hunk 5]
```diff
@@ -4,32 +4,65 @@ title: Tracking
...
+
+## Tracking underway
+
+Once a tracking session has been setup, a badge will appear in the side dashboard
+legend next to the layer(s) against which tracking is being recorded.
+
+!![](../assets/images/tracking-badge.png)
+
+The feature(s) created by the tracking session is/are saved on every vertex recorded.
```
Comment (m-kuhn) [inline documentation/how-to/tracking.en.md:49]:
```suggestion
The features created by the tracking session are saved on every vertex recorded.
```
Easier to read and correct too

File: documentation/how-to/tracking.en.md (hunk 4 above)
Comment (nirvn) [reply documentation/how-to/tracking.en.md:40]: @m-kuhn , we'll have to make this true by the time we publish 2.3 :)

**Review by m-kuhn — APPROVED** [review]

File: documentation/how-to/tracking.en.md (hunk 4 above)
Comment (m-kuhn) [reply documentation/how-to/tracking.en.md:40]: It's already respected for simple feature creation IIRC

Comment (nirvn) [reply documentation/how-to/tracking.en.md]: @m-kuhn , it is now a truth: https://github.com/opengisch/QField/pull/3247/commits/e1e98bb94bb034f48f6850b32865c785485ac9a4 :)

**Pull request merged**